.github/workflows/release.hutch-agent.yml

```diff
@@ -0,0 +1,83 @@
+name: Release Manager (stable)
+
+on:
+  workflow_dispatch: # manual trigger
```

Comment:
while we've got this PR open, and we've discovered we can do the following

```yaml
on:
  pull_request:
    types: [closed]
    branches: [release/**]

jobs:
  if_merged:
    if: github.event.pull_request.merged == true
    # ...
```

(see docs [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request) for combining type with compare branch and [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges) for triggering on PRs when they merge.)

should we amend this one to use that trigger approach before merging this PR?